src/coprocessor_v2/config.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    pub coprocessor_plugin_directory: String,
+}
+
+impl Default for Config {
+    fn default() -> Config {
+        Config {
+            coprocessor_plugin_directory: "coprocessors".to_string(),
```

Comment:
What's the difference between a default value here and the value in `etc/config-template.toml`? Are they related or are they independent?